README.md

```diff
@@ -54,7 +54,9 @@ Your result will look like:
 ...
 ```
 
-**think**: Why does 03/13 come out as _sad_ when it should be _happy_? How could we fix this?
+**think**: Why does 03/13 come out as _neutral_ when it should be _happy_? How could we fix this?
+
+We could look for key words like yesterday and today. Also I'm guessing that yay with an exclamation point won't count as included in the happy array.
```

Comment:
What does looking for key words like yesterday and today help us? Correct on the ! :)